contributing.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Contributing
+---
+
+## I think I found a bug:
+Submit an [issue](https://github.com/watir/watir-webdriver/issues) or [pull request](https://github.com/watir/watir-webdriver/pulls) to the [watir-webdriver project](https://github.com/watir/watir-webdriver).
+
+## I want to contribute to the documentation:
+The public webpage is maintaned as a GitHub page written in Jekyll. To contribute, send a PR at [our webpage](https://watir.github.io). 
```

Comment:
Should the link at the end of this line go to the github repo instead of back to the homepage?